server/server_controller.py

```diff
@@ -13,6 +16,14 @@ def create_server(self, name, host, port):
             'address': None
         }
 
+    def create_default(self, name):
+        s = Server(name, socket.gethostbyname(socket.gethostname()), random.randint(2000, 50000))
+        self.servers[name] = {
+            'server': s,
+            'connection': None,
+            'address': None
+        }
+
```

Comment:
The method is very similar to create_server. Maybe we can send a parameter indicating the type of server it will create.

Reply:
Revise now